panic: assignment to entry in nil map during installation

RELEEM_MYSQL_MEMORY_LIMIT=0 RELEEM_API_KEY=* RELEEM_MYSQL_ROOT_PASSWORD='' RELEEM_CRON_ENABLE=1 bash -c "$(curl -L https://releem.s3.amazonaws.com/v2/install.sh)"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 19603  100 19603    0     0  33357      0 --:--:-- --:--:-- --:--:-- 33338

 * Installing dependencies...


 * Downloading Releem Agent, architecture x86_64...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 18170  100 18170    0     0  34693      0 --:--:-- --:--:-- --:--:-- 34741
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 17.5M  100 17.5M    0     0  10.4M      0  0:00:01  0:00:01 --:--:-- 10.4M

 * Configure the application...

 * Detected service name for appling config

 * Configure catalog for copy recommend config

 * The /etc/my.cnf file is used for automatic Releem settings.

 * Adding directive includedir to the MySQL configuration /etc/my.cnf.

 * Configure MySQL user for collect data

 * Using MySQL root user

 Connect to MySQL - successful

 Created new user `releem`

 Connecting to MySQL with user `releem` - successfull

 * Configure mysql memory limit

 * Saving variables to Releem Agent configuration

 - Adding API key to the Releem Agent configuration: /opt/releem/releem.conf
 - Adding Releem Configuration Directory /opt/releem/conf to Releem Agent configuration: /opt/releem/releem.conf
 - Adding user and password mysql to the Releem Agent configuration: /opt/releem/releem.conf
 - Adding MySQL restart command to the Releem Agent configuration: /opt/releem/releem.conf
 - Adding MySQL include directory to the Releem Agent configuration /opt/releem/releem.conf.

 * Configure crontab...

 * Executing Releem Agent for first time...
INFO     : [Main] - 2023/07/25 05:46:37 main.go:82: Starting releem-agent of version is 1.5.0.3
INFO     : [Main] - 2023/07/25 05:46:37 config.go:40: Loading config /opt/releem/releem.conf
INFO     : [Main] - 2023/07/25 05:46:37 main.go:150: InstanceType is Local
INFO     : [Main] - 2023/07/25 05:46:37 main.go:179: Connect Success to DB via tcp 127.0.0.1
INFO     : [Worker] - 2023/07/25 05:46:37 runner.go:71:  * Collecting metrics to recommend a config...
panic: assignment to entry in nil map

goroutine 5 [running]:
http://github.com/Releem/mysqlconfigurer/metrics.MapJoin(...)
        /opt/mysqlconfigurer/metrics/Metrics.go:67
http://github.com/Releem/mysqlconfigurer/metrics.(*DbMetricsGatherer).GetMetrics(0xc0003ce100, 0xc0000bc000)
        /opt/mysqlconfigurer/metrics/dbMetrics.go:107 +0x106d
http://github.com/Releem/mysqlconfigurer/metrics.collectMetrics({0xc0000000a0, 0x6, 0x0?}, {0xda9470, 0xc0003d6d70})
        /opt/mysqlconfigurer/metrics/runner.go:98 +0xf1
http://github.com/Releem/mysqlconfigurer/metrics.RunWorker.func2()
        /opt/mysqlconfigurer/metrics/runner.go:72 +0x1f1
created by http://github.com/Releem/mysqlconfigurer/metrics.RunWorker
        /opt/mysqlconfigurer/metrics/runner.go:69 +0x88c
INFO     : [Main] - 2023/07/25 05:46:43 main.go:82: Starting releem-agent of version is 1.5.0.3
INFO     : [Main] - 2023/07/25 05:46:43 config.go:40: Loading config /opt/releem/releem.conf
INFO     : [Main] - 2023/07/25 05:46:43 main.go:150: InstanceType is Local
INFO     : [Main] - 2023/07/25 05:46:43 main.go:179: Connect Success to DB via tcp 127.0.0.1
INFO     : [Worker] - 2023/07/25 05:46:46 runner.go:47: Exiting

 * Installing and starting Releem Agent service to collect metrics..

 Installing Releem Agent - successful

 Restarting Releem Agent - successful

 * Enabling Performance schema and SlowLog to collect metrics...


 * Performance schema and SlowLog are enabled for metrics collection.

 * Releem Agent is successfully installed.

 * To view Releem recommendations and MySQL metrics, visit https://app.releem.com/dashboard

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -2,6 +2,24 @@ Releem releases
 ---
 Information about releases of the Releem.
 
+Releem 1.7.0, 2023-08-31 ([What's New At Releem | August 2023](https://releem.com/blog/whats-new-at-releem-august-2023))
+- Added Automated Updates for Releem Agent installed in docker container. Closes #184
+- Improved version for mobile and Firefox compatibility.
+- Improved Query Cache suggestions. Closes #135
+- Fixed 'innodb_max_dirty_pages_pct' bug on MySQL 5.5. Closes #183
+- Fixed metrics collecttion issue for db servers with Sphinx engine. Closes #174 , Closes #175
+- Fixed bug for users with unapproved email. Closes #179
+- Fixed bug with saving errors in MySQL configuration when Releem Platform reply with errors.
+
+Releem 1.6.0, 2023-07-31 ([What's New At Releem | July 2023](https://releem.com/blog/whats-new-at-releem-july-2023))
+- Added IOPS graph to Releem Dashboard.
+- Added System Information to System Metrics block. Closes #169
+- Improved MySQL Metrics graph and included ‘Applying Configuration’ events on the timeline.
+- Improved graphs and made Y-axis absolute and not relative. Closes #167
+- Removed innodb_flush_log_at_trx_commit automatic recommendations. Closes #170
+- Fixed minor issues with the 'innodb_buffer_pool_instance' and 'thread_cache' MySQL variables.
+- Improved Releem Agent installation for older MySQL and MariaDB versions without Performance Schema.
+
 Releem 1.5.0, 2023-06-30 ([What's New At Releem | June 2023](https://releem.com/blog/whats-new-at-releem-june-2023))
 - Improved the Recommended configuration window to show users all variables that Releem tunes and details on variables. 
 - Improved MySQL metric charts with buttons and avg metrics remove Latency and SlowLog

--- a/README.md
+++ b/README.md
@@ -11,9 +11,9 @@
 </p>
 <p align="center">
     <a href="https://releem.com/docs/getstarted">Docs<a> | 
-    <a href="https://releem.com/rds">AWS RDS</a> |
-    <a href="https://releem.com/blog">Blog</a> |
-    <a href="https://releem.com/compare/mysqltuner">Compare MySQLTuner</a>
+    <a href="https://releem.com/compare/mysqltuner">Compare MySQLTuner</a> |
+    <a href="https://releem.com/sql-query-optimization">SQL Query Optimization</a> |
+    <a href="https://releem.com/blog">Blog</a>
     <br /><br />
 </p>
 
@@ -64,6 +64,7 @@ Join the Releem Community on [Slack](https://join.slack.com/t/releem-community/s
 - MariaDB 10.1, MariaDB 10.2, MariaDB 10.3, MariaDB 10.4, MariaDB 10.5, MariaDB 10.6, MariaDB 10.7, MariaDB 10.8, MariaDB 10.9, MariaDB 10.10, MariaDB 10.11, MariaDB 11.0
 - Percona 8.0, Percona 5.7, Percona 5.6, Percona 5.5
 - Centos, CloudLinux, Debian, Ubuntu, RockyLinux
+- Amazon RDS MySQL, Amazon RDS Aurora
 
 *** MINIMAL REQUIREMENTS ***
 - Unix/Linux based operating system (tested on Linux, BSD variants, and Solaris variants)

--- a/README.md
+++ b/README.md
@@ -64,7 +64,7 @@ Join the Releem Community on [Slack](https://join.slack.com/t/releem-community/s
 - MariaDB 10.1, MariaDB 10.2, MariaDB 10.3, MariaDB 10.4, MariaDB 10.5, MariaDB 10.6, MariaDB 10.7, MariaDB 10.8, MariaDB 10.9, MariaDB 10.10, MariaDB 10.11, MariaDB 11.0
 - Percona 8.0, Percona 5.7, Percona 5.6, Percona 5.5
 - Centos, CloudLinux, Debian, Ubuntu, RockyLinux
-- Amazon RDS MySQL, Amazon RDS Aurora
+- Amazon RDS MySQL, Amazon RDS Aurora, Amazon RDS MariaDB
 
 *** MINIMAL REQUIREMENTS ***
 - Unix/Linux based operating system (tested on Linux, BSD variants, and Solaris variants)

